ElectronClient/plugins/GotoAnything.jsx

```diff
@@ -64,7 +64,7 @@ class Dialog extends React.PureComponent {
 		const itemHeight = this.state.resultsInBody ? 84 : 64;
 
 		this.styles_[styleKey] = {
-			dialogBox: Object.assign({}, theme.dialogBox, { minWidth: '50%', maxWidth: '50%' }),
+			dialogBox: Object.assign({}, theme.dialogBox, { minWidth: '50%', maxWidth: '50%', maxHeight: '80%', display: 'flex', flexDirection: 'column' }),
```

Comment:
Could you instead apply the fix to theme.dialogBox? (in theme.js) That way all dialog boxes will benefit from it. Also in that case, please check the other dialog boxes to make sure they still work.

Reply:
Sure sir, I will do it.